### PR TITLE
added folders homework

### PR DESCRIPTION
**Pull request by lesia-mk**

### CODE REVIEW

**Comment by lesia-mk** [comment]
image
image
image

File: HW2/table and form.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    </head>
+<body>
+    <h1>Form</h1>
+    <form action="#" name="firstForm" method="post">
+
+        <fieldset>
+            <legend>Basic info</legend>
+            <p>
+                <label for="name">ID</label>
+                <input id="name" type="text" value="325">
```
Comment (spolischook) [inline HW2/table and form.html]: readonly or disabled attribute should be applied

File: HW2/table and form.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                <label for="firstName">First Name</label>
+                <input id="firstName" type="text">
+            </p>
+            <p>
+                <label for="lastName">Last Name</label>
+                <input id="lastName" type="text">
+            </p>
+            <p>
+                <label for="password">Password</label>
+                <input id="password" type="password" value="123456789">
```
Comment (spolischook) [inline HW2/table and form.html]: remove value attribute it's prohibited here

File: HW2/table and form.html [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+                <textarea name="text" id="about"></textarea>
+            </p>
+            <p>
+                <label for="profile">Profile picture</label>
+                <input type="file" id="profile" name="photo">
+            </p>
+        </fieldset>
+    </form>
+
+    <form action="#" name="secondForm" method="post">
```
Comment (spolischook) [inline HW2/table and form.html]: form should be one on the page in context of this HW

File: HW2/table and form.html [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            <p>
+                <input type="radio" id="mid" name="interest" value="mid-l">
+                <label for="mid">Mid-Level Developer</label>
+            </p>
+            <p>
+                <input type="radio" id="senior" name="interest" value="senior-d">
+                <label for="senior">Senior Developer</label>
+            </p>
+            <p>Country
+                <select name="select">
```
Comment (spolischook) [inline HW2/table and form.html]:
```suggestion
                <select name="country">
```

File: HW2/table and form.html [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+                    <option disabled selected>-Choose your country-</option>
+                    <option value="ua">Ukraine</option>
```
Comment (spolischook) [inline HW2/table and form.html:92]: :+1:

File: HW2/table and form.html (hunk 4 above)
Comment (spolischook) [inline HW2/table and form.html]: Preferred choices is missed. See wire-frame

File: HW2/table and form.html [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+                    <option value="sp">Spain</option>
+                    <option value="usa">USA</option>
+                    <option value="pl">Poland</option>
+                </select>
+            </p>
+            <p>Skills:
+            </p>
+            <p>
+                <input type="checkbox" name="a" value="html" checked>HTML
```
Comment (spolischook) [inline HW2/table and form.html]:
```suggestion
                <input type="checkbox" name="skills" value="html" checked>HTML
```

File: HW2/table and form.html [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+                <input type="checkbox" name="a" value="r">R
+            </p>
+            <p>
+                <input type="checkbox" name="a" value="scala">Scala
+            </p>
+            <p>
+                <input type="checkbox" name="a" value="haskell">Haskell
+            </p>
+            <p>
+                <input type="hidden" id="author" name="author" value="Lesia">
```
Comment (spolischook) [inline HW2/table and form.html:120]: :+1:

**Review by spolischook — CHANGES_REQUESTED** [review]
```name``` attribute is missed for fields

I've suggest use ```div``` as wrapper for form fields instead of ```p```, however Yana suggest use ```li``` and I'm agree, because it's semantically right

File: HW2/table and form.html [hunk 8]
```diff
@@ -7,66 +7,62 @@
...
-    <form action="#" name="firstForm" method="post">
-
+    <form action="https://postman-echo.com/post" method="post">
         <fieldset>
             <legend>Basic info</legend>
             <p>
-                <label for="name">ID</label>
-                <input id="name" type="text" value="325">
+                <label for="id">ID</label>
+                <input id="id" name="id"type="text" value="325">
```
Comment (spolischook) [inline HW2/table and form.html]: don't forget about ```readonly``` or ```disabled``` attribute

File: HW2/table and form.html [hunk 9]
```diff
@@ -12,7 +12,7 @@ <h1>Form</h1>
             <legend>Basic info</legend>
             <p>
                 <label for="id">ID</label>
-                <input id="id" name="id"type="text" value="325">
+                <input id="id" name="id"type="text" value="325" readonly>
```
Comment (spolischook) [inline HW2/table and form.html:15]: :sun_with_face:

**Pull request merged**